ilan/Dockerfile.template

```diff
@@ -0,0 +1,34 @@
+############################################################################
+# (C) Copyright IBM Corporation 2015.                                      #
+#                                                                          #
+# Licensed under the Apache License, Version 2.0 (the "License");          #
+# you may not use this file except in compliance with the License.         #
+# You may obtain a copy of the License at                                  #
+#                                                                          #
+#      http://www.apache.org/licenses/LICENSE-2.0                          #
+#                                                                          #
+# Unless required by applicable law or agreed to in writing, software      #
+# distributed under the License is distributed on an "AS IS" BASIS,        #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. #
+# See the License for the specific language governing permissions and      #
+# limitations under the License.                                           #
+#                                                                          #
+############################################################################
+
+# This is a template Dockerfile which will have version of the tar expected
+# in the COPY command set during the previous docker build.
+
+FROM ubuntu:16.04
+
+MAINTAINER Kavitha Suresh Kumar <kavisuresh@in.ibm.com>
+
+RUN apt-get update && apt-get install --no-install-recommends -y unzip bsdtar && rm -rf /var/lib/apt/lists/*
+# BSD tar is need to prevent errors on Docker builds
+# see https://github.com/docker/hub-feedback/issues/727
+RUN export tar='bsdtar'
+
+COPY ihsstart.sh /work/
+COPY im/ihs@VERSION@.tar.gz /ihs.tar.gz
```

Comment:
This step means that you end up with the .tar.gz in a layer in the final image even though it is hidden by the layer above. That's why the current approach serves up the files so that they can be pulled in, unpacked and then deleted all as part of the same layer.